scaling_performance/device_plugin.adoc

```diff
@@ -0,0 +1,133 @@
+[[scaling-performance-using-device-plugin]]
+= Using the Device Plug-in
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[what-the-device-plugin-does]]
+== What the Device Plug-in Does
```

Comment:
This part will be replaced with https://github.com/openshift/openshift-docs/pull/7751/files#diff-a0529a66faa63bb82338644ecf45eec5